Hubble Space Telescope updates + Bugfixes

MeuhV1 opened the pull request:
Corrected Hubble Space Telescope pedia text
Hubble Space Telescope has space environment in the pedia

Commit:
Corrected Hubble Space Telescope pedia text
Hubble Space Telescope has space environment in the pedia

MeuhV1 commented:
Commit 2:
Bugfix in RFCUtils
Exported ChangeBuildingYieldChange to python. (Required for the Space Elevator and probably some other wonders/projects as well)
Spaceship screen doesn't automatically open when a space project is finished.

dguenms commented on CvGameCoreDLL/CvCity.cpp:
```diff
@@ -14415,38 +14415,39 @@ void CvCity::popOrder(int iNum, bool bFinish, bool bChoose)
 					{
 						CvPopupInfo* pInfo = NULL;
 
-						if (GC.getGameINLINE().isNetworkMultiPlayer())
```
Can you use multiline comments? Like this:
```
/* This is
a multiline
comment*/
```

dguenms commented on CvGameCoreDLL/CvCity.cpp:
```diff
@@ -14415,38 +14415,39 @@ void CvCity::popOrder(int iNum, bool bFinish, bool bChoose)
-						}
-					}
-				}
+				// else
```
Same here.

dguenms reviewed (approved):
Ready to merge after rebase.

Commit:
Bugfix in RFCUtils
Exported ChangeBuildingYieldChange to python (required for Space Elevator and probably some other wonders/projects)
Spaceship screen doesn't automatically open when a space project is constructed

MeuhV1 commented:
Done.
